docs/en/ingest-management/commands.asciidoc

```diff
@@ -761,6 +763,12 @@ Install {agent} in a non-interactive mode. This flag is helpful when
 using automation software or scripted deployments. If {agent} is
 already installed on the host, the installation will terminate.
 
+`--privileged`::
```

Comment:
This (and it's counterpart) should become commands: `elastic-agent privileged` and`elastic-agent unprivileged`.

Or you got to know `--privileged` exist(?)

Reply:
For now I've included both the commands (`elastic-agent privileged`, `elastic-agent unprivileged`) and the install command flags (`elastic-agent install --privileged`, `elastic-agent install --unprivileged`).